api/src/opentrons/protocol_engine/commands/unsafe/unsafe_place_labware.py

```diff
@@ -129,9 +129,11 @@ async def execute(
                     module.id
                 )
 
-        # NOTE: When the estop is pressed, the gantry loses position,
-        # so the robot needs to home x, y to sync.
-        await ot3api.home(axes=[Axis.Z_L, Axis.Z_R, Axis.Z_G, Axis.X, Axis.Y])
+        # NOTE: When the estop is pressed, the gantry loses position, lets use
+        # the encoders to sync position.
```

Comment:
I think it would be worth a comment here for why we're doing this careful selective home thing.

```suggestion
        # NOTE: When the estop is pressed, the gantry loses position, lets use
        # the encoders to sync position.
        # Ideally, we'd do a full home, but this command is used when
        # the gripper is holding the plate reader, and a full home would
        # bang it into the right window.
```